pkg/interface/src/views/apps/chat/components/chat-editor.js

```diff
@@ -142,19 +165,38 @@ export default class ChatEditor extends Component {
         paddingTop='8px'
         width='calc(100% - 88px)'
         className={inCodeMode ? 'chat code' : 'chat'}
+        color="black"
       >
-        <CodeEditor
+        {MOBILE_BROWSER_REGEX.test(navigator.userAgent)
+          ? <input
```

Comment:
you can import `BaseInput` from indigo-react for this.

custom.css had logic for whether `code` descended elements would pass Source Code Pro; now Code or Message doesn't change font face on this one, because that declaration is too specific. the BaseInput component could have `mono={inCodeMode}` to compensate. It also should have `fontSize='14px'` to match the chat input, in which case you'll want to ensure it's lined up properly with the sigil, etc.

Reply:
`mono` doesn't seem to work on `BaseInput`...

Reply:
ohh right I guess that's specific to `Text`. You could do similar with `fontFamily` probably.